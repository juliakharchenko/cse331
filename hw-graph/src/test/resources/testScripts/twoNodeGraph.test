# Tests if multiple nodes can be added to the graph

# Create a graph
CreateGraph multiNodeGraph

# Add nodes
AddNode multiNodeGraph a
AddNode multiNodeGraph b

# Print the nodes in the graph and the outgoing edges from a and b
ListNode multiNodeGraph
ListChildren multiNodeGraph a
ListChildren multiNodeGraph b