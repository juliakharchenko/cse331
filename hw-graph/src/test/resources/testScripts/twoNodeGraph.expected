# Tests if multiple nodes can be added to the graph

# Create a graph
created graph multiNodeGraph

# Add nodes
added node a to multiNodeGraph
added node b to multiNodeGraph

# Print the nodes in the graph and the outgoing edges from a and b
multiNodeGraph contains: a b
the children of a in multiNodeGraph are:
the children of b in multiNodeGraph are: